gui/default/assets/css/overrides.css

```diff
@@ -312,11 +312,7 @@ a.toggler:hover {
     text-decoration: none;
 }
 
-/**
- * Panel padding decrease
- */
-
-.panel-collapse .panel-body {
+.panel-body.less-padding {
```

Comment:
This change should probably be separated out into a new PR on the main repo.  It touches many files and makes the PR needlessly bigger.

Reply:
Ok, see https://github.com/syncthing/syncthing/pull/9659